First_project/First_project/AuxiliaryElements/AppConstants.swift

```diff
@@ -8,8 +8,20 @@
 import UIKit
 
 enum AppConstants{
+    static let imageHeigh: CGFloat = imageWidth
+    static let imageWidth: CGFloat = 420
```

Comment:
Image width and height should be equal to screen width. It can be different for iPhone 13pro max and iPhone 6